<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherConfigWidget</class>
 <widget class="QDialog" name="WeatherConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Weather Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="m_informationGroupBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="m_weatherConditionBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;\np, li { white-space: pre-wrap; }\n&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Here you can choose if you want the weather condition to be shown (for example, clouds or clear sky.) Marble will only show the condition if this information is available.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
</string>
        </property>
        <property name="text">
         <string>Weather condition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_temperatureBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;\np, li { white-space: pre-wrap; }\n&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Selects if you want the temperature to be shown. The temperature will only be shown for the stations it is available for.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
</string>
        </property>
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_windDirectionBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;\np, li { white-space: pre-wrap; }\n&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select if you want the wind direction to be shown. The wind direction will only be shown if it is available for the particular station.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
</string>
        </property>
        <property name="text">
         <string>Wind direction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_windSpeedBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;\np, li { white-space: pre-wrap; }\n&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select if you want the wind speed to be shown. The wind speed will only be shown if it is available for the particular station.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
</string>
        </property>
        <property name="text">
         <string>Wind speed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_unitsGroupBox">
     <property name="title">
      <string>Units</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="m_temperatureLabel">
          <property name="text">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_temperatureComboBox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;\np, li { white-space: pre-wrap; }\n&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The unit used to measure the temperature.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
</string>
          </property>
          <item>
           <property name="text">
            <string>Celsius</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fahrenheit</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="m_windSpeedLabel">
          <property name="text">
           <string>Wind speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_windSpeedComboBox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;\np, li { white-space: pre-wrap; }\n&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The unit used to measure the wind speed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
</string>
          </property>
          <item>
           <property name="text">
            <string>Kilometers per hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Miles per hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Meters per second</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Knots</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Beaufort</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="m_pressureLabel">
          <property name="text">
           <string>Pressure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_pressureComboBox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;\np, li { white-space: pre-wrap; }\n&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The unit used to measure the air pressure.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
</string>
          </property>
          <item>
           <property name="text">
            <string>Hecto Pascals</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kilo Pascals</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bars</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Millimeters of Mercury</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inches of Mercury</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_miscGroupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="m_onlyFavoritesBox">
        <property name="text">
         <string>Show only favorite items</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
        <item>
         <widget class="QLabel" name="m_updateIntervalLabel">
          <property name="text">
           <string>Weather update interval</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_updateIntervalBox">
          <property name="suffix">
           <string> hours</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>80</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WeatherConfigWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WeatherConfigWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
